pytorch_lightning/tuner/lr_finder.py

```diff
@@ -62,24 +61,160 @@ def _determine_lr_attr_name(trainer, model: LightningModule) -> str:
     )
 
 
+class _LRFinder(object):
+    """ LR finder object. This object stores the results of Trainer.lr_find().
+
+    Args:
+        mode: either `linear` or `exponential`, how to increase lr after each step
+
+        lr_min: lr to start search from
+
+        lr_max: lr to stop search
+
+        num_training: number of steps to take between lr_min and lr_max
+
+    Example::
+        # Run lr finder
+        lr_finder = trainer.lr_find(model)
```

Comment:
```suggestion
        lr_finder = trainer.tuner.lr_find(model)
```

Reply:
Good catch, I'll update this in the next PR (#7258) which changes how to call these functions